step-framework-parent/step-framework-timeseries/src/main/java/step/core/timeseries/metric/MetricTypeRenderSettings.java

```diff
@@ -0,0 +1,51 @@
+package step.core.timeseries.metric;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This class contains all needed settings for rendering a visual chart.
+ */
+public class MetricTypeRenderSettings {
+    
+    private String groupingAttribute; // the default grouping
+    private List<String> groupingOptions; // in case we will allow the user to change the grouping like Google does.
+    private Map<String, String> seriesColors; // can set predefined colors for known series.
+    private String yAxesUnit; // e.g. '/hour' or 'ms' displayed both on the tooltip and on the y axes
```

Comment:
The unit of the metric should be part of the MetricType class and should be renamed "unit"

Reply:
ok